cfpb/consumerfinance.gov
# Bump raft version to 0.4.0.

Updates version of rental assistance finder tool to 0.4.0. Users were finding it confusing when they could not locate their county in the dropdown for their state, so this version shows all counties (using data derived from a Census data set) rather than just counties represented in the program data.

More information on the release can be found here: https://github.com/cfpb/rental-assistance-finder/releases/tag/0.4.0

---



## Changes

- Update raft version


## How to test this PR

1. Check out branch and run yarn gulp scripts
2. Navigate to [RAFT page](http://localhost:8000/coronavirus/mortgage-and-housing-assistance/renter-protections/find-help-with-rent-and-utilities/)
3. Select California in the state select and verify that county dropdown shows all counties
4. Verify that selecting Alpine shows 1 result
5. Verify that selecting Alameda shows 4 results


## Screenshots
Select    | Result
 --- | ---
 California&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; | <img width="783" alt="Screen Shot 2021-10-04 at 10 03 06 AM" src="https://user-images.githubusercontent.com/778171/135893510-a85f6b5b-3928-4e9a-9881-1b0c3e0398e4.png">
Alameda | <img width="784" alt="Screen Shot 2021-10-04 at 10 00 16 AM" src="https://user-images.githubusercontent.com/778171/135893159-72c4357b-3d33-4e0f-9805-d4e52e807a37.png">
Alpine | <img width="788" alt="Screen Shot 2021-10-04 at 10 00 23 AM" src="https://user-images.githubusercontent.com/778171/135893153-3e5cb6d5-d382-4cc7-ab0a-772f78c55a46.png">
